### ISSUE TITLE
[1.16] Config does not reload on save

### ISSUE THREAD

**Issue by legobmw99** [yuitakaa/InvTweaksReborn#35]
Changing the rules and saving the config file does not affect the sorting until restart

**Comment by galyfray** [comment]
yeah that's normal it's the same thing with all mods config are loaded at startup

**Comment by legobmw99** [comment]
It is possible to reload configs while running the game, and the mod[ intends to do so](https://github.com/reo-ar/InvTweaksRenewed/blob/1.16.x/src/main/java/invtweaks/config/InvTweaksConfig.java#L221-L225), this functionality is just not working (or wasn't at the time of the report)
